cmd/util.go

```diff
@@ -0,0 +1,18 @@
+package cmd
+
+import (
+	"github.com/redhat-developer/odo/pkg/component"
+	"github.com/redhat-developer/odo/pkg/occlient"
+)
+
+// getComponent returns the component to be used for the operation. If an input
+// component is specified, then it is returned, if not, the current component
+// is fetched and returned
+func getComponent(client *occlient.Client, inputComponent, applicationName, projectName string) string {
+	if len(inputComponent) == 0 {
+		c, err := component.GetCurrent(client, applicationName, projectName)
+		checkError(err, "Could not get current component")
+		return c
+	}
+	return inputComponent
```

Comment:
It would be good to check if the component exists here, so we don't have to do it each time we call this function.

(we need to check it only if `len(inputComponent)!=0`

Reply:
added